pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/TransformFunctionFactory.java

```diff
@@ -198,6 +202,18 @@ public static TransformFunction get(ExpressionContext expression, Map<String, Da
     return get(null, expression, dataSourceMap);
   }
 
+  public static ExpressionContext stripGapfill(ExpressionContext expression) {
```

Comment:
I feel this util method does not belong to this class. Suggest adding a util class for gapfill, and put all the shared constants and methods there

Reply:
Done